webui/react/vite.config.mts

```diff
@@ -89,7 +90,7 @@ export default defineConfig(({ mode }) => ({
     'process.env.IS_DEV': JSON.stringify(mode === 'development'),
     'process.env.PUBLIC_URL': JSON.stringify((mode !== 'test' && publicUrl) || ''),
     'process.env.SERVER_ADDRESS': JSON.stringify(process.env.SERVER_ADDRESS),
-    'process.env.VERSION': '"0.32.1-dev0"',
+    'process.env.VERSION': JSON.stringify(process.env.VERSION || child.execSync('git describe --tags --always 2>/dev/null || echo "unknown"').toString().trimEnd()),
```

Comment:
i ran `git describe --tags --always 2>/dev/null || echo "unknown"` in the determined root dir, and i got `0.0.102-3487-gb9d11f69325`. it doesnt looks like the latest tag. is it expected?

Reply:
Yup, you're seeing the correct behavior, albeit an interesting case of it, because that string implies that 0.0.102 is the last reachable version tag in your repository, and that you're 3,487 commits ahead of it, on commit `b9d11f69325`. I don't have `0.0.102` in my local repository anywhere, so I'm guessing it's local to yours. But it's working correctly, yes.